src/modules/we-track/controllers/we-track.controller.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
```

Comment:
missing a space

Reply:
Oh wow, you'll have to take that up here:
https://github.com/nestjs/nest/issues

Reply:
Actually looks like they all do that